### PR TITLE
added cipher suite details

### PR DESCRIPTION
**Pull request by javabeanz**
added easy getters for key exchange, authorizatoin, algorythm and mac method.

### CODE REVIEW

File: src/main/java/com/mps/deepviolet/suite/json/CipherSuite.java [hunk 1]
```diff
@@ -3,8 +3,43 @@
 public class CipherSuite {
 
 	private String name;
+	private String keyExchange;
+	private String authentication;
+	private String algorythm;
```
Comment (kingthorin) [inline src/main/java/com/mps/deepviolet/suite/json/CipherSuite.java]: I think you want `algorithm` here (and throughout).

Comment (spoofzu) [reply src/main/java/com/mps/deepviolet/suite/json/CipherSuite.java]: @javabeanz :  Yes, I'm with @kingthorin if you change methods to algorithm I will go ahead and merge the changes.  Sorry, I know I have been sitting on this for awhile.

Comment (javabeanz) [reply src/main/java/com/mps/deepviolet/suite/json/CipherSuite.java]: I can see that the new OWASP board members election is in progress. Good luck Milton !

Comment (spoofzu) [reply src/main/java/com/mps/deepviolet/suite/json/CipherSuite.java]: It's all up to OWASP members and who they want for their board at this point.  Go ahead and change the  "algorythm" references to "algorithm".  I feel bad about dragging my feet on this for so long.

Comment (kingthorin) [reply src/main/java/com/mps/deepviolet/suite/json/CipherSuite.java]: @javabeanz any news on addressing that issue?

**Comment by spoofzu** [comment]
This has been hanging around long enough, closing.

**Comment by kingthorin** [comment]
@spoofzu do you want the changes re-created? I can submit a PR that addresses the comment...

**Pull request closed without merging**